### PR TITLE
New package: aws-cli-1.16.155

### LINKED ISSUE #11435: Package request: aws-cli
https://github.com/aws/aws-cli (Python based, Apache-2.0 licensed)

### PR DESCRIPTION
**Pull request by bobertlo**
closes #11435

### CODE REVIEW

File: srcpkgs/python-botocore/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	vlicense LICENSE.txt
+}
+
+python3-botocore_package() {
+	archs=noarch
+	pycompile_module="botocore"
+	depends="python3-dateutil python3-docutils python3-jmespath python3-urllib3"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
```
Comment (Hoshpak) [inline srcpkgs/python-botocore/template]: This should be quoted

File: srcpkgs/python-s3transfer/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	vlicense LICENSE.txt
+}
+
+python3-s3transfer_package() {
+	archs=noarch
+	depends="python3-botocore"
+	pycompile_module="s3transfer"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
```
Comment (Hoshpak) [inline srcpkgs/python-s3transfer/template]: same here

**Comment by bobertlo** [comment]
@Hoshpak ok those are fixed

**Comment by Hoshpak** [comment]
Oh, I meant just the argument, not the whole command. Like `vmove "usr/lib/python3*"`

**Comment by bobertlo** [comment]
I need to stop using my laptop in bed. I'm amazed. I'll fix this as soon as
I'm around a keyboard again!

**Comment by bobertlo** [comment]
@Hoshpak got it. Still not sure how that happened :)

**Pull request merged**